### ISSUE TITLE
Custom settings are not set in PdfGlobal

### ISSUE THREAD

**Issue by bornste** [tuespetre/TuesPechkin#122]
I derived from the GlobalSettings class to have some more properties exposed:

``` C#
public class PdfGlobalSettings : GlobalSettings
{
    [WkhtmltoxSetting("viewportSize")]
    public string ViewportSize { get; set; }

    [WkhtmltoxSetting("resolveRelativeLinks")]
    public bool? ResolveRelativeLinks { get; set; }
}
```

Unfortunately that does not work and logging tells me for both of them:

```
pechkin:default Information: 0 : T: Setting global setting 'viewportSize' to '1280x1024' for config 147526992
pechkin:default Information: 0 : ...setting was not successful
pechkin:default Information: 0 : T: Setting global setting 'resolveRelativeLinks' to 'true' for config 147526992
pechkin:default Information: 0 : ...setting was not successful
```

I tried it with the `Win64EmbeddedDeployment` (0.12.2.1) and the `StaticDeployment` (0.12.2.4 binaries) without success.

Beside that everything works like a charm.

What am I missing here? Any help is appreciated.

Thank you, Stefan

**Comment by tuespetre** [comment]
Neither of those properties are reflected for use in the API as of v0.12.2.4. See: https://github.com/wkhtmltopdf/wkhtmltopdf/blob/0.12.2.4/src/lib/pdfsettings.cc#L107

On master branch, 'resolveRelativeLinks' is reflected but not 'viewportSize'. You may consider opening an issue or submitting a pull request to them. 
- Derek Gray

**Comment by bornste** [comment]
I've opened an issue here:
https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2609

Thanks Derek

**Comment by stephenrichardson** [comment]
This issue has been resolved: [wkhtmltopdf/wkhtmltopdf#2609](https://github.com/wkhtmltopdf/wkhtmltopdf/issues/2609)
Can we get viewportSize added to a new release please?
